[OSDOCS#16946]: Make changes as per the errors and warnings for CQA-2 (#114626)
openshift/openshift-docs: modules/persistent-storage-fibre-provisioning.adoc

Reviewer: This looks more like a reference than concept

//
// * storage/persistent_storage/persistent-storage-fibre.adoc

:_mod-docs-content-type: REFERENCE
[id="provisioning-fibre_{context}"]

> = Provisioning

Reviewer 2: Not a procedure
```suggestion
= About provisioning storage using Fibre Channel
```

[role="_abstract"]

> To provision Fibre Channel volumes using the `PersistentVolume` API the following must be available.

Reviewer 2: ```suggestion
[role="_abstract"]
You can provision Fibre Channel volumes by using the `PersistentVolume` API 

Before provisioning the volumes, the following items must be available.
```

> * Fibre Channel LUNs must exist in the underlying infrastructure.

Reviewer 2: I don't think this is appropriate in a concept, unless you want to convert this to a procedure. And, you moved the bullet to a note.

```suggestion

```

    - ReadWriteOnce
  fc:
    wwids: [3600508b400105e210000900000490000]
    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5']
    lun: 2
    fsType: ext4
----
where:

`spec.fc.wwids`:: Specifies the world wide identifiers (WWIDs). Either FC `wwids` or a combination of FC `targetWWNs` and `lun` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. The WWID identifier can be obtained by issuing a SCSI Inquiry to retrieve the Device Identification Vital Product Data (`page 0x83`) or Unit Serial Number (`page 0x80`). FC WWIDs are identified as `/dev/disk/by-id/` to reference the data on the disk, even if the path to the device changes and even when accessing the device from different systems.

`spec.fc.targetWWNs`:: Specifies the Fibre Channel World Wide Names (WWNs). Fibre Channel WWNs are identified as `/dev/disk/by-path/pci-<IDENTIFIER>-fc-0x<WWN>-lun-<LUN#>`, but you do not need to provide any part of the path leading up to the `WWN`, including the `0x`, and anything after, including the `-` (hyphen).

> `spec.fc.targetWWNs.lun`:: Specifies the LUN number.

Reviewer: ```suggestion
`spec.fc.lun`:: Specifies the LUN number.
```
